kernel: allow seamless migration from I40EVF

peterwillcn opened the pull request:
Resolve the issue of seamless migration from I40EVF

link: https://github.com/openwrt/openwrt/pull/19197/files#r2173571237
Reported-and-Tested-by: Stefan Hellermann <[stefan@the2masters.de](mailto:stefan@the2masters.de)>

Signed-off-by: xiao bo <peterwillcn@gmail.com>

the2masters reviewed (approved):
Reported-and-Tested-by: Stefan Hellermann <<stefan@the2masters.de>>

peterwillcn committed:
kernel: allow seamless migration from I40EVF

Resolve the issue of seamless migration from I40EVF
link: https://github.com/openwrt/openwrt/pull/19197/files#r2173571237

Signed-off-by: xiao bo <peterwillcn@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19255
Signed-off-by: Robert Marko <robimarko@gmail.com>
